src/Umbraco.Web.BackOffice/Controllers/UsersController.cs

```diff
@@ -785,22 +785,29 @@ public IActionResult PostDisableUsers([FromQuery] int[] userIds)
             return ValidationProblem("The current user cannot disable itself");
         }
 
-        IUser[] users = _userService.GetUsersById(userIds).ToArray();
+        var users = _userService.GetUsersById(userIds).ToList();
+        List<IUser> skippedUsers = new();
         foreach (IUser u in users)
         {
+            if (u.UserState is UserState.Invited)
+            {
+                _logger.LogWarning("Could not disable invited user {Username}", u.Name);
+                skippedUsers.Add(u);
+                continue;
+            }
+
             u.IsApproved = false;
             u.InvitedDate = null;
         }
 
-        _userService.Save(users);
+        users = users.Where(x => !skippedUsers.Contains(x)).ToList();
```

Comment:
```suggestion
        users = users.Except(skippedUsers).ToList();
```

Reply:
I didn't know that was a thing! You learn something everyday with LINQ 😆